src/styles/reset.css

```diff
@@ -0,0 +1,9 @@
+body,
+h3,
+p {
+  margin: 0;
+}
+
+.html {
+  font-family: Roboto, sans-serif;
+}
```

Comment:
Do we need to add this to the 'reset.css' file? I think it should be placed in the card.css.